dev/io.openliberty.xmlws.3.0.internal.tools/resources/com/ibm/ws/jaxws/tools/internal/resources/JaxWsToolsMessages.nlsprops

```diff
@@ -0,0 +1,26 @@
+###############################################################################
+# Copyright (c) 2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+
+# NLS_MESSAGEFORMAT_ALL
+#
+#CMVCPATHNAME com.ibm.ws.jaxws.ejb/resources/com/ibm/ws/jaxws/tools/internal/resources/JaxWsToolsMessages.nlsprops
```

Comment:
Similar to the classes, you should consider changing the package of the nls files as well.  Additionally the CMVCPATHNAME is clearly wrong.  Additionally the JAX-WS references probably should be changed to XML WS.

Reply:
I've refactored the target option requirement out of the code, and deleted these because their only purpose seemed to be forcing the target option to be specified, which while the target option seems is still supported doesn't result in a behavior change as this tooling only supports 3.0.

Reply:
To me more clear, I don't see a reason to force a requirement to specify a target value for specs that no longer matter prior to 3.0.